inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/group/InlongGroupServiceImpl.java

```diff
@@ -498,6 +523,7 @@ public String update(InlongGroupRequest request, String operator) {
 
         // save schedule info for offline group
         if (DATASYNC_OFFLINE_MODE.equals(request.getInlongGroupMode())) {
+            constrainStartAndEndTime(request);
```

Comment:
checkStartAndEndTime

Reply:
I think constrain is better here since we do check and re-set both